scripts/registry.sh

```diff
@@ -91,6 +87,17 @@ publish() {
       | (popd > /dev/null && git hash-object -w --stdin))
 
     git commit -am "chore: update versions"
+
+    while ! yarn lerna publish from-package \
+      --dist-tag="$DISTTAG" --no-git-reset --no-verify-access --yes; do
+      echo 1>&2 "Retrying publish..."
+      sleep 5
+    done
+    git reset --hard HEAD
+
+    # Convention used in Endo
+    yarn lerna run clean:types
```

Comment:
this isn't necessary anymore https://github.com/search?q=repo%3Aendojs%2Fendo%20clean%3Atypes&type=code